app/src/main/java/edu/uc/app/parkfinder/ui/main/MainFragment.kt

```diff
@@ -24,4 +20,8 @@ class MainFragment : Fragment(R.layout.main_fragment) {
                 parks -> actPark.setAdapter(ArrayAdapter(requireContext(), R.layout.support_simple_spinner_dropdown_item, parks))
         })
     }
+
+    companion object {
```

Comment:
Move companion object to end of class (Kotlin best practices)

Reply:
This duplicates a previous code review.  When that occurs, first in wins.